docs/reference/generated/kubeadm.md

```diff
@@ -111,8 +111,8 @@ flags that can be used to customise the Kubernetes installation.
 - `--apiserver-advertise-address`
 
   This is the address the API Server will advertise to other members of the
```

Comment:
Replace this entire paragraph (lines 113-116) with:
```
This is the address the API Server will advertise to other members of the cluster.  The API server IP address can either be IPv4 or IPv6. This is also the address used to construct the suggested `kubeadm join` line at the end of the init process.  If not set (or set to `0.0.0.0` for IPv4 or `::` for IPv6) then the IP for the default interface will be used.
```
So:
> This is the address the API Server will advertise to other members of the cluster.  The API server IP address can either be IPv4 or IPv6. This is also the address used to construct the suggested `kubeadm join` line at the end of the init process.  If not set (or set to `0.0.0.0` for IPv4 or `::` for IPv6) then the IP for the default interface will be used.

Reply:
This replacement ☝️ includes @luxas's feedback.

Reply:
@mmueen Please add the requested syntax by copying and pasting the following section:
```
This is the address the API Server will advertise to other members of the cluster.  The API server IP address can either be IPv4 or IPv6. This is also the address used to construct the suggested `kubeadm join` line at the end of the init process.  If not set (or set to `0.0.0.0` for IPv4 or `::` for IPv6) then the IP for the default interface will be used.
```